vars/packageBuildingPipeline.groovy

```diff
@@ -42,6 +42,20 @@
 //project to point to the above branch.  Then build and test as usual
 
 def call(Map pipeline_args) {
+    if (!pipeline_args) {
+        pipeline_args = [:]
+    }
+    if (pipeline_args['distros']) {
+        distros = pipeline_args['distros']
+    } else {
+        distros = ['centos7', 'sles12.3', 'leap42.3',
+                   'leap15', 'ubuntu18.04', 'ubuntu_rolling']
```

Comment:
I think we should drop ubuntu18.04 from the default distros.

Reply:
Done